app-ios/Sources/TimetableFeature/TimetableListView.swift

```diff
@@ -13,18 +13,33 @@ public struct TimetableView: View {
     
     @State var timetableMode = TimetableMode.list
     @State var switchModeIcon: String = "square.grid.2x2"
+    @State var selectedTab: DayTab = DayTab.day1
     
     public var body: some View {
         VStack {
             HStack {
                 ForEach(DayTab.allCases) { tabItem in
                     Button(action: {
                         store.send(.view(.selectDay(tabItem)))
+                        selectedTab = tabItem
                     }, label: {
                         //TODO: Only selected button should be green and underlined
-                        Text(tabItem.rawValue).foregroundStyle(
-                            AssetColors.Custom.flamingo.swiftUIColor)
-                            .underline()
+                        if selectedTab == tabItem {
+                            HStack(spacing: 6) {
+                                Text(tabItem.rawValue)
+                                    .textStyle(.labelMedium)
+                                    .underline()
+                            }
+                            .foregroundStyle(AssetColors.Custom.iguana.swiftUIColor)
+                            .padding(6)
+                        } else {
+                            HStack(spacing: 6) {
+                                Text(tabItem.rawValue)
+                                    .textStyle(.labelMedium)
+                            }
+                            .foregroundStyle(AssetColors.Surface.onSurface.swiftUIColor)
+                            .padding(6)
+                        }
```

Comment:
I think following code is better because it is short and clear what is changed👀 
```suggestion
                        HStack(spacing: 6) {
                                Text(tabItem.rawValue)
                                    .textStyle(.labelMedium)
                                    .underline(selectedTab == tabItem)
                            }
                            .foregroundStyle(selectedTab == tabItem ? AssetColors.Custom.iguana.swiftUIColor : AssetColors.Surface.onSurface.swiftUIColor)
                            .padding(6)
```